testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1662,3 +1714,8 @@
   node = get_target(target)
   node.run("iptables -F INPUT")
 end
+
+When(/^I restart the "([^"]*)" on "([^"]*)"$/) do |service, minion|
```

Comment:
```suggestion
When(/^I restart the "([^"]*)" service on "([^"]*)"$/) do |service, minion|
```

Reply:
Before or after the double quotes, that I don't mind, but I mean, making easier to understand that we are restarting a service.

Reply:
Fixed.